### PR TITLE
Add a basic PR Review document

### PR DESCRIPTION
**Pull request by cmaloney**
cc: @spahl @lingmann

### CODE REVIEW

File: docs/pr-reviews.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# PR Reviews
+
+There are two different kinds of PRs: code / individual change PRs and trains. Individual PRs are pulled into a train, where they are tested together, bad / conflicting prs can be kicked off the train, then the whole train lands at once. Ocassionally an individual train is merged wihtout a train, but most the time trains are used to make it so everything doesn't have to be rebased all the time to guarantee it all actually works together.
```
Comment (orsenthil) [inline docs/pr-reviews.md]: Spell check plz. For e.g. wihtout, Ocassionally 

```
There are two different kinds of PRs: code change PRs and trains. Code change PRs are pulled into a train, where they tested together,  conflicting PRs can be removed from the train, then the whole train lands at once. Occasionally an individual train is merged without a train, but most the time trains are used to make it, so everything doesn't have to be rebased all the time to guarantee it all works together.```

File: docs/pr-reviews.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## General PR Reviews (WIP)
+
+WIP list of Things to Verify:
+1. all tests are passing (things with non-passing tests won't get reviewed, if a author requests will do basic comment path about the general direction or helping fix why tests don't pass)
```
Comment (orsenthil) [inline docs/pr-reviews.md]: Caps All. `All tests should pass`.

File: docs/pr-reviews.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+2. Code is pythonic
+3. Doesn't introduce a lot of complexity / is as simple as possible
+4. All systemd changes follow [Systemd Rules](systemd-rules.md)
+5. All package changes follow [Package Rules](package-rules.md)
+
+
+## Merge Train PR Reviews
+
+Things to verify
+1. All the PRs referenced have :ship: from me / seb / albert / jeremy after the last change
```
Comment (orsenthil) [inline docs/pr-reviews.md]: Full stop after this sentence and include `@cmaloney` in the statement.

File: docs/pr-reviews.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+2. The github message includes all the PRs
+3. a quick high level review that there isn't anything i don't expect in there
```
Comment (orsenthil) [inline docs/pr-reviews.md]: Capitalize and mention it as "owner merging the train" instead of "i".

**Comment by lingmann** [comment]
My main feedback is can we adjust this so that this is more understandable by the community? As written it feels very much like this will only be understood by those currently maintaining DC/OS. Would suggest a little intro sentence that describes who this document is for and a link to the doc community members should read if they want to submit a PR.

File: docs/pr-reviews.md (hunk 2 above)
Comment (amitaekbote) [inline docs/pr-reviews.md]: Mention tox?

File: docs/pr-reviews.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# PR Reviews
+
```
Comment (amitaekbote) [inline docs/pr-reviews.md]: Before PR section? We can add links to github-pr-labels, systemd-rules and tox testing. We are assuming they know all that when they are reading the rest

**Comment by cmaloney** [comment]
Closing for now, wil ressucitate when there's actually time to wokr on this.

**Pull request closed without merging**